hasura/migrations/1553873750399_update_webhook_project.up.sql

```diff
@@ -0,0 +1,2 @@
+alter table projects
```

Comment:
hmm the default should be a uuid 
so when we get a summ.hub/organization_id/webhook_id
so we can match it up with DB,